lib/rubocop/cop/layout/leading_blank_lines.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Layout
+      # This cop checks for unnecessary leading blank lines at the beginning
+      # of a file.
+      #
+      # @example
+      #
+      # # bad
+      # # SOF
```

Comment:
I wasn't sure how to best represent the beginning of a file here. I opted for SOF, Start of File. Please let me know if anyone has a better recommendation.

Reply:
I cannot think of representation examples other than this writing method 💦 , but I can understand this examples well.

As a little worrisome point, `# SOF` may be better for `# BOF` (Beginning Of File).
Also, I'm not sure, but `# EOF` may be unnecessary in this examples. Because this `Layout/LeadingBlankLines` cop checks for unnecessary leading blank lines at the beginning of file. This allows the examples readers to focus on the differences in SOF (BOF) behavior.

Perhaps the examples of `Layout/TrailingBlankLines` cop may be a similar reference.
https://github.com/rubocop-hq/rubocop/blob/v0.56.0/lib/rubocop/cop/layout/trailing_blank_lines.rb#L6-L39

Reply:
I think we can just expand those comments to `# (start of file)`, `# (blank line)`, etc. I guess people will figure out the need for comments in the examples.

Reply:
@Drenmi I think it's always nice to have some code example just to be on the safe side. I don't like `SOF` much because that's also the airport code for my hometown (Sofia, Bulgaria). 😆